app/controllers/api/v1/products_controller.rb

```diff
@@ -19,15 +19,15 @@ def create
     if product.save
       render json: ProductSerializer.render(product), status: :created
     else
-      render_error(4130, product.errors.messages[:error])
+      render_error(4200, product.errors.messages[:error])
     end
   end
 
   def update
     if product.update(product_params)
         render json: ProductSerializer.render(product)
       else
-        render_error(4130, product.errors.messages[:error])
+        render_error(4201, product.errors.messages[:error])
       end
```

Comment:
identation seems to be wrong (?)
Also you could instead do:  return render ... on the if body and avoid the else, just putting the render_error as last line